src/feature/control/control_getinfo.c

```diff
@@ -1513,6 +1551,8 @@ static const getinfo_item_t getinfo_items[] = {
          "v2 networkstatus docs as retrieved from a DirPort."),
   ITEM("dir/status-vote/current/consensus", dir,
        "v3 Networkstatus consensus as retrieved from a DirPort."),
+  ITEM("dir/status-vote/current/consensus-microdesc", dir,
```

Comment:
Please consider adding a unittest for the new functionality. See how `test_download_status_consensus()` does it for example.